e2e/terraform/variables.tf

```diff
@@ -87,6 +87,12 @@ variable "hcp_vault_namespace" {
   default     = "admin"
 }
 
+variable "aws_kms_alias" {
+  description = "The alias for the AWS KMS key ID"
+  type        = string
+  default     = "kms-nomad-keyring"
```

Comment:
just checking: if I turn on my own cluster for my own purposes, I can leave this as-is and not collide with any other in-flight e2e clusters, right?  like I don't need to go out of my way to create a new kms key (which is done in our other nomad-e2e repo) and point this at it?

Reply:
So long as you don't also change the IAM instance profile, yes that'll work just fine. The KMS is only used to encrypt/decrypt and doesn't store the actual key material at all.